index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML>
+<html lang="en" ng-app="todoList">
+  <head>
+    <meta charset="utf-8">
+    <title>To-Do List</title>
+    <script src="bower_components/jquery/dist/jquery.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-resource/angular-resource.js"></script>
+    <script src="public/js/todoController.js"></script>
+  </head>
+
+<body ng-controller="todoController">
+  <h1>To-Do List:</h1>
+
+  <form id="add-form">
+    <input type="text" ng-model="newTask.text" placeholder="Add something to the list!">
+    <button id="new_task" ng-click="add()">Add item</button>
+  </form>
+  
+  <div >
+    <ul>
+      <li ng-repeat="text in allTasks">
```

Comment:
I think the variable you're using in this ng-repeat should be 'task' rather than 'text' so that it's more semantic. What's being repeated is the task object as a whole (both the text and the completion status), so then you can use mustaches to access {{task.text}} or {{task.done}} (at least in principle, I know this is still breaking for some strange reason)